cmd/mimir/help-all.txt.tmpl

```diff
@@ -1145,6 +1145,12 @@ Usage of ./cmd/mimir/mimir:
     	Per-tenant allowed push request burst size. 0 to disable.
   -distributor.request-rate-limit float
     	Per-tenant push request rate limit in requests per second. 0 to disable.
+  -distributor.retry.base duration
+    	[experimental] Base time to wait before retrying a 429/5xx request. (default 3s)
+  -distributor.retry.enabled
+    	[experimental] Enabled controls inclusion of the Retry-After header in the response: true includes it for client retry guidance, false omits it.
+  -distributor.retry.max-allowed-attempts int
+    	[experimental] Sets the upper limit on the number of retry attempts considered for calculation. It caps the calculated attempts without rejecting additional attempts. Used for controlling exponential backoff calculations. (default 5)
   -distributor.ring.consul.acl-token string
```

Comment:
can you also mention these in `about-versioning.md`? Some short description about how to use this feature

Reply:
about-versiong.md updated